package.json

```diff
@@ -50,6 +51,7 @@
     "build": "run build",
     "build:clean": "run prepare",
     "docs": "run docs \"@theia/!(example-)*\"",
+    "docs-modules": "rimraf gh-pages/docs/next && typedoc --readme README.MD --name 'Theia TypeDoc' --exclude '**/+(dev-packages|examples|typings)/**/*.ts' --options configs/typedoc.json --module commonjs --target es6 --hideGenerator --mode modules --external-modulemap '.*\/packages\/([\\w\\-_]+)\/' --out gh-pages/docs/next",
```

Comment:
So this will regenerate the typedoc for our packages, whenever we publish a `next` version. 

The "official" releases (`latest`) are done manually. Should we update the doc for those releases too? If so it would be nice to update [Publishing.md](https://github.com/theia-ide/theia/blob/master/doc/Publishing.md) to give instructions how.

Reply:
ok will provide instructions